src/stim/py/compiled_detector_sampler.pybind.cc

```diff
@@ -108,12 +108,14 @@ CompiledDetectorSampler stim_pybind::py_init_compiled_detector_sampler(
     return CompiledDetectorSampler(circuit, make_py_seeded_rng(seed));
 }
 
-pybind11::class_<CompiledDetectorSampler> stim_pybind::pybind_compiled_detector_sampler_class(pybind11::module &m) {
+pybind11::class_<CompiledDetectorSampler> stim_pybind::pybind_compiled_detector_sampler(pybind11::module &m) {
     return pybind11::class_<CompiledDetectorSampler>(
         m, "CompiledDetectorSampler", "An analyzed stabilizer circuit whose detection events can be sampled quickly.");
 }
 
-void stim_pybind::pybind_compiled_detector_sampler_methods(pybind11::class_<CompiledDetectorSampler> &c) {
+void stim_pybind::pybind_compiled_detector_sampler_methods(
+    pybind11::module &,
```

Comment:
Huh, I didn't know you could omit names of unused params.

Reply:
oh yeah. I was considering doing this but decided against it. Just missed this instance